### PR TITLE
docs(chore) revision on calc-constant doc

### PR DESCRIPTION
**Pull request by bsmth**
Hi Ramiy, I've made a revision of your changes to make it a bit more concise 👍🏻

### CODE REVIEW

File: files/en-us/web/css/calc-constant/index.md [hunk 1]
```diff
@@ -14,55 +14,45 @@ browser-compat: css.types.calc-constant
...
+
   - : The ratio of a circle's circumference to its diameter, approximately equal to `3.1415926535897932`.
 
 - `infinity` & `-infinity`
+
   - : An infinite value, used to indicate the largest/smallest possible value.
 
 - `NaN`
-  - : A value representing "Not a Number" canonical casing.
+  - : A value representing "Not a Number". This constant case-sensitive.
```
Comment (wbamberg) [inline files/en-us/web/css/calc-constant/index.md]:
```suggestion
  - : A value representing "Not a Number". This constant is case-sensitive.
```

File: files/en-us/web/css/calc-constant/index.md [hunk 2]
```diff
@@ -14,55 +14,45 @@ browser-compat: css.types.calc-constant
...
+The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.
+These constants are supported for convenience in CSS so that authors do not have to manually write or calculate them.
+Mathematical constants can only be used inside [CSS math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) for calculations.
+If used outside of a calculation, they're treated like any other keyword.
+For example:
+
+- `animation-name: pi;` refers to an animation named "pi", not the `pi` numeric constant.
+- `line-height: e;` is invalid, but `line-height: calc(e);` is valid.
+
+Using `infinity` or `-infinity` is intended to be more convenient for authors to create large fake rather than calculating or generating them through other means.
```
Comment (wbamberg) [inline files/en-us/web/css/calc-constant/index.md]: large fake? Not sure what this means :).

File: files/en-us/web/css/calc-constant/index.md [hunk 3]
```diff
@@ -14,55 +14,45 @@ browser-compat: css.types.calc-constant
...
+Although `infinity` and `NaN` are not technically numbers, they are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example.
```
Comment (wbamberg) [inline files/en-us/web/css/calc-constant/index.md]: I think the bit about "they are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example" is true about all these constants, so it is misleading to hitch it to "Although `infinity` and `NaN` are not technically numbers".

I don't really think "Although `infinity` and `NaN` are not technically numbers" is helpful. What does "not technically numbers" mean?

```suggestion
In math functions, `<calc-constant>` values are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example.
```

File: files/en-us/web/css/calc-constant/index.md [hunk 4]
```diff
@@ -14,55 +14,45 @@ browser-compat: css.types.calc-constant
...
 {{CSSRef}}
 
-The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents well-defined constants such as `e` and `π`. Rather than require authors to manually type out several digits of these constants, a few of them are provided directly by CSS.
+The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.
+These constants are supported for convenience in CSS so that authors do not have to manually write or calculate them.
+Mathematical constants can only be used inside [CSS math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) for calculations.
+If used outside of a calculation, they're treated like any other keyword.
+For example:
+
+- `animation-name: pi;` refers to an animation named "pi", not the `pi` numeric constant.
```
Comment (wbamberg) [inline files/en-us/web/css/calc-constant/index.md:23]: I'm happy with this, ish, although I like the practice of having a very short summary at the start:

> The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.

...and then putting the other stuff in an H2 Description that goes after H2 Syntax.

File: files/en-us/web/css/calc-constant/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/web/css/calc-constant/index.md]: 😂 I meant to write something along the lines of "large dummy / mock data". I'll have another go...

File: files/en-us/web/css/calc-constant/index.md (hunk 4 above)
Comment (bsmth) [reply files/en-us/web/css/calc-constant/index.md:23]: That makes sense, I'm happy with either

Comment (ramiy) [reply files/en-us/web/css/calc-constant/index.md:23]: > I like the practice of having a very short summary at the start

Me too.

**Comment by bsmth** [comment]
I'm closing this and will apply changes directly to https://github.com/ramiy/content/tree/patch-5

**Pull request closed without merging**